Readme.md

```diff
@@ -0,0 +1,8 @@
+# ROV
+
+## ros2_ws
+This is the ROS2 workspace for the ROV. It contains the following packages:
+- joy_controller - A package that will take in joystick input and publish it to the /joy topic
+- **TODO** - create basic file structure for the rest of the packages
+
```

Comment:
Add a `## Getting Started` section with some instructions (i.e. run colcon build in the ros2_ws folder. Downgrade setuptools if needed, how to test install is working, sourcing the `setup.sh` file, etc).

Reply:
assigned in trello